community-content/prediction_featurestore_integration/prediction_featurestore_integration.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>Maybe better to display the actual request body here.</p><p> </p><p>Consider simplifying the construction of the predict request.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>